src/index.ts

```diff
@@ -39,7 +40,18 @@ class TypingEffectElement extends HTMLElement {
     }
   }
 
+  get prefersReducedMotion(): boolean {
+    if (this.getAttribute('data-reduced-motion') === 'false') {
```

Comment:
The intention with this attribute is allow folks to opt-out of the default behavior for users with `prefers-reduced-motion` enabled, right? It's a little confusing that `data-reduced-motion = true` does nothing. Do you think an attribute like `data-ignore-reduced-motion = true` or `data-force-effect = true` (or something else entirely!) could be less confusing?

Reply:
Hmmm i think that's a better way to phrase / use the attribute. I like the idea of being able to set reduced motion both true and false through the attribute, so I'll allow that `data-reduced-motion = true` will force `prefersReducedMotion` to return `true`.

So basically we are saying that `data-reduced-motion` overrides the value of any window media query if it's set to `true` or `false`, right?